### ISSUE TITLE
Duplicate Function configure_stackpower_stack Error

### ISSUE THREAD

**Issue by abemarji** [CiscoTestAutomation/genielibs#131]
Function definition for **configure_stackpower_stack** in configure.py file is defined twice and generating Error when calling API.

```
def configure_stackpower_stack(device, powerstack_name, mode=None, strict=False):
```

Problem also is applicable to unconfigure function:  
```
def unconfigure_stackpower_stack(device, powerstack_name):
```

**Location of the Issue:**   
```
genielibs/pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxe/platform/configure.py
```

**Screen Shot:**
<img width="344" alt="image" src="https://github.com/CiscoTestAutomation/genielibs/assets/33726075/50703f28-4cc6-4a4d-aa75-ac8233917af2">

**Comment by abemarji** [comment]
Errors generated when using API with RobotFramework Script:
```
[ ERROR ] Error in library 'genie.libs.robot.GenieRobotApis': Adding keyword 'Configure Stackpower Stack' failed: Keyword with same name defined multiple times.
[ ERROR ] Error in library 'genie.libs.robot.GenieRobotApis': Adding keyword 'Unconfigure Stackpower Stack' failed: Keyword with same name defined multiple times.
```

**Comment by SohanTirpude** [comment]
Hello @abemarji,

Currently I am working on this issue. Once I have any update on it, I'll update you here. Kindly give me some time.

**Comment by SohanTirpude** [comment]
Hello @abemarji, the PR has been raised for this issue. Once it is merged, I'll update you here. Thank you.

**Comment by SohanTirpude** [comment]
Hello @abemarji,

The PR got merged and the fix will be available in upcoming pyATS version 23.10.

Hence, I am closing this ticket. If you face same issue with pyATS 23.10, then kindly let me know, I'll reopen the ticket. If you face any other issues, kindly open a new ticket. Thank you.
